src/tests/functional/navigation_tests.ts

```diff
@@ -116,6 +116,22 @@ export class NavigationTests extends TurboDriveTestCase {
     this.assert.equal(await this.pathname, "/src/tests/fixtures/one.html")
     this.assert.equal(await this.visitAction, "restore")
   }
+
+  async "test skip link with hash-only path"() {
+    await this.clickSelector('a[href="#main"]')
+    await this.nextBeat
```

Comment:
Do we need `this.nextBeat`? I'm thinking we shouldn't be reloading the page so I don't think this is needed.

Reply:
Hmm. Removing it doesn't cause failures as it should. Is there a way to test that a page _hasn't_ reloaded. e.g. checking that a Turbo event hasn't fired?